packages/connectivity/pubspec.yaml

```diff
@@ -3,8 +3,10 @@ description: Flutter plugin for discovering the state of the network (WiFi &
   mobile/cellular) connectivity on Android and iOS.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/connectivity
+
 version: 0.4.3+4
```

Comment:
please remember to update the pubspec and changelog to a new version after everything is done.